src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/DefaultTexts/DefaultTexts.test.tsx

```diff
@@ -43,10 +43,8 @@ describe('<DefaultTexts />', () => {
       <DefaultTexts {...props} />
     )
 
-    expect(queryAllByTestId('defaultTextsTitle')).toHaveLength(1)
-    expect(queryByTestId('defaultTextsTitle')).toHaveTextContent(
-      /^Standaardtekst$/
-    )
+    expect(queryAllByTestId('modalTitle')).toHaveLength(1)
+    expect(queryByTestId('modalTitle')).toHaveTextContent(/^Standaardtekst$/)
```

Comment:
This should be `getByTestId` and then the previous line can be omitted

Reply:
✅